### PR TITLE
Add sample data and data generators

### LINKED ISSUE #148: Add sample data for production

### PR DESCRIPTION
**Pull request by kantoniak**
* Adds sample groups, users, cafeterias and menu entries
* Adds generator scripts for camera events and reviews
* Remove old data dump

Cafeteria occupancy is combined from:
* Uniformly distributed noise around the day
* Peak around lunchtime (all enter around peak time with some Gaussian distribution)

Closes #148.

### CODE REVIEW

File: django-server/README.md [hunk 1]
```diff
@@ -54,4 +54,16 @@ python manage.py dumpdata --exclude=auth --exclude=contenttypes --exclude=sessio
...
+
+## Sample data
+
+To load sample data in empty project, run:
+```
+python manage.py migrate
+python manage.py loaddata fixtures/demo-data.yaml
+python manage.py reset_passwords  # Set password equal to username for each user
+python manage.py generate_events  # Use --from/to=YYYY-MM-DD to select dates
+python manage.py generate_reviews  # Use --from/to=YYYY-MM-DD to select dates
```
Comment (avan1235) [inline django-server/README.md:67]: great

File: django-server/README.md [hunk 2]
```diff
@@ -54,4 +54,16 @@ python manage.py dumpdata --exclude=auth --exclude=contenttypes --exclude=sessio
...
\ No newline at end of file
+```
+
+## Sample data
+
+To load sample data in empty project, run:
+```
+python manage.py migrate
+python manage.py loaddata fixtures/demo-data.yaml
+python manage.py reset_passwords  # Set password equal to username for each user
```
Comment (avan1235) [inline django-server/README.md:65]: not sure why do we need this and if it should be available for such system

Comment (kantoniak) [reply django-server/README.md:65]: Here I added it just so you can add user by hand and don't have to run all hashing etc. It's not a problem in terms of security: if you have shell access, then you can do the same thing by hand using `python manage.py shell`.

Generally, Django passwords are salted. If salt uses configured security key, then you can't guarantee your local passwords will be the same as in prod: https://docs.djangoproject.com/en/3.0/topics/auth/passwords/#how-django-stores-passwords

**Comment by kantoniak** [comment]
Regarding data dumps, I chose YAML for readability. It's much easier to edit by hand andfile has ~50 times less lines than JSON dump.

**Comment by kantoniak** [comment]
Regarding sample, I chose YAML for readability. It's much easier to edit demos by hand and file has ~50 times less lines than JSON dump.

**Pull request merged**